src/test/java/hudson/plugins/gradle/WithGradleWorkFlowTest.java

```diff
@@ -44,19 +47,16 @@
     private String build_gradle = "writeFile(file:'build.gradle', text:'defaultTasks \\\'hello\\\'\\ntask hello << { println \\\'Hello\\\' }') \n";
 
     /*
-        TODO DO NOT MERGE: when jenkins-test-harness-tools 2.1 is released add GradleInstallation tool. Right now
-        these tests use the system's default Gradle. If Gradle is not installed these tests will fail.
-
-        TODO: convert these tests to Groovy OR create src/test/java/.../WithGradleWorkFlowTest.java
         TODO? add test for console annotation
         TODO? add test for reloading annotator on Jenkins restart
     */
 
     @Test
     public void testStepDefaultTools() throws Exception {
+        String name = ToolInstallations.configureDefaultGradle(new TemporaryFolder()).getName();
```

Comment:
I actually would prefer that the Gradle plugin itself would publish a test fixture so that other plugins can depend on it and the fixture itself is more adjusted to the needs of Gradle. Even the [test-harness-tools](https://github.com/jenkinsci/jenkins-test-harness-tools) state that this should be the strategy for the future:
> *New versions are unreleasable*. Alternatives to consider:
>
> - allowing plugins to specify test-scoped dependencies on tool zips, with some utility to extract those in jenkins-test-harness, and an extension handler in maven-hpi-plugin allowing them to be added to the classpath
> - use docker-fixtures to connect to agents running well-defined versions of various tools

I will not reject merging this PR if this is the only problem. We should consider that in the future.